trestle/core/commands/author/jinja.py

```diff
@@ -68,6 +68,12 @@ def _init_arguments(self):
             help='Add incremental numbering to table and image captions, in the form Table n - ... and Figure n - ...',
             action='store_true'
         )
+        self.add_argument(
+            '-pf',
+            '--param-formatting',
+            help='Add given text to each parameter, use dot to specify location of param relative to text. i.e. [ . ].',
+            required=False
```

Comment:
I would ignore grammar and leave off the period at the end of that text.  Is there a better short example to use, like foo.bar will wrap the parameter with foo and bar?